src/deleteLexicon.js

```diff
@@ -0,0 +1,28 @@
+import AWS from 'aws-sdk';
+import Syncano from 'syncano-server';
+import helper from './util/helper';
+
+export default (ctx) => {
+  const { response, logger } = Syncano(ctx);
+
+  const log = logger('Socket scope');
+  const pollyHelper = new helper(ctx.config);
+
+  const deleteLexicon = pollyHelper.deleteLexicon(ctx.args.lexiconName);
+
+  return deleteLexicon
```

Comment:
you can do `return pollyHelper.deleteLexicon(ctx.args.lexiconName).then(...`

Reply:
done